src/create_helpers.ts

```diff
@@ -107,6 +115,34 @@ export function createClass(doclet: IClassDoclet, children?: ts.Node[]): ts.Clas
         );
     }
 
+    if (doclet.properties)
+    {
+        const nodes = (new PropTree(doclet.properties)).roots;
```

Comment:
I prefer this read as `const tree = new PropTree(doclet.properties)` and then you use `tree.roots` instead of `nodes` below.